lib/components-schema-v1_6_x.ts

```diff
@@ -0,0 +1,600 @@
+                        enum: ['no', 'yes', 'one-level'],
+                        description: 'Allows nesting the component in containers, defaults to "yes"',
+                    },
+                    restrictChildren: {
+                        type: 'object',
+                        description:
+                            'Restricts children of this component to the listed ones and can be further filtered down to also have content',
+                        minProperties: 1,
+                        additionalProperties: {
+                            type: 'object',
+                            properties: {
+                                withContent: { type: 'string' },
+                            },
+                            additionalProperties: false,
+                        },
+                    },
+                    countStatistics: {
+                        type: 'boolean',
+                        description: 'Count characters, words and paragraphs of this component',
+                    },
+                    defaultComponentOnEnter: {
+                        $ref: '#/definitions/nonEmptyString',
+                        description:
+                            'Default component inserted on pressing enter in a text field (optional, overrides global value)',
+                    },
+                    directiveOptions: {
+                        type: 'object',
+                        description: 'Configuration for directives in this component',
+                        minProperties: 1,
+                        additionalProperties: {
+                            type: 'object',
+                            additionalProperties: false,
+                            properties: {
+                                groups: componentGroupDefinition,
+                                autofill: {
+                                    type: 'object',
+                                    additionalProperties: false,
+                                    required: ['source'],
+                                    properties: {
+                                        source: {
+                                            type: 'string',
+                                            description: 'A key of source directive',
+                                        },
+                                        metadataField: {
+                                            type: 'string',
+                                            description:
+                                                'Metadata property name. The field is case sensitive. Slashes should be used to separate levels',
+                                        },
+                                    },
+                                },
+                                stripStylingOnPaste: {
+                                    type: 'boolean',
+                                    description: 'Strip all styling when pasting into the directive.',
```

Comment:
'Strip all styling when pasting into an "editable" directive'? I believe we also don't end the descriptions with periods in the definition file in case of a single sentence, but seems inconsistent :(